### PR TITLE
feat(planning_launch): add obstacle_cruise_planner

### PR DESCRIPTION
**Pull request by takayuki5168**
## PR Type

- New Feature

## Description

launch PR for autoware.universe PR https://github.com/autowarefoundation/autoware.universe/pull/570, adding obstacle_velocity_planner

## Pre-Review Checklist for the PR Author

**PR Author should check the checkboxes below when creating the PR.**

- [ ] Code follows [coding guidelines][coding-guidelines]
- [ ] Assign PR to reviewer

## Checklist for the PR Reviewer

**Reviewers should check the checkboxes below before approval.**

- [ ] Commits are properly organized and messages are according to the guideline
- [ ] Code follows [coding guidelines][coding-guidelines]
- [ ] (Optional) Unit tests have been written for new behavior
- [ ] PR title describes the changes

## Post-Review Checklist for the PR Author

**PR Author should check the checkboxes below before merging.**

- [ ] All open points are addressed and tracked via issues or tickets
- [ ] Write [release notes][release-notes]

## CI Checks

- **Build and test for PR**: Required to pass before the merge.
- **Check spelling**: NOT required to pass before the merge. It is up to the reviewer(s). See [here][spell-check-dict] if you want to add some words to the spell check dictionary.

[coding-guidelines]: https://tier4.atlassian.net/wiki/spaces/AIP/pages/1194394777/T4
[release-notes]: https://tier4.atlassian.net/wiki/spaces/AIP/pages/563774416
[spell-check-dict]: https://github.com/tier4/autoware-spell-check-dict#how-to-contribute

### CODE REVIEW

File: planning_launch/launch/scenario_planning/lane_driving/motion_planning/motion_planning.launch.py [hunk 1]
```diff
@@ -168,10 +201,21 @@ def generate_launch_description():
...
+        composable_node_descriptions=[obstacle_stop_planner_component],
+        target_container=container,
+        condition=IfCondition(LaunchConfiguration("use_obstacle_stop_planner")),
+    )
+
+    obstacle_velocity_planner_loader = LoadComposableNodes(
+        composable_node_descriptions=[obstacle_velocity_planner_component],
+        target_container=container,
+        condition=IfCondition(LaunchConfiguration("use_obstacle_velocity_planner")),
+    )
```
Comment (satoshi-ota) [inline planning_launch/launch/scenario_planning/lane_driving/motion_planning/motion_planning.launch.py:237]: [imo]
https://github.com/autowarefoundation/autoware.universe/pull/570#discussion_r885099649

Comment (takayuki5168) [reply planning_launch/launch/scenario_planning/lane_driving/motion_planning/motion_planning.launch.py:237]: https://github.com/tier4/autoware_launch/blob/385b0e58e2b653e05e16328fb58fe74d705e3bec/planning_launch/launch/scenario_planning/lane_driving/motion_planning/motion_planning.launch.py#L263-L265
DONE

File: planning_launch/launch/scenario_planning/lane_driving/motion_planning/motion_planning.launch.xml [hunk 2]
```diff
@@ -27,7 +27,7 @@
     <include file="$(find-pkg-share obstacle_stop_planner)/launch/obstacle_stop_planner.launch.xml">
       <!-- load config files -->
       <arg name="common_param_path" value="$(var common_param_path)"/>
-      <arg name="adaptive_cruise_control_param_path" value="$(find-pkg-share planning_launch)/config/obstacle_stop_planner/adaptive_cruise_control.param.yaml" />
+      <arg name="obstacle_cruise_control_param_path" value="$(find-pkg-share planning_launch)/config/obstacle_stop_planner/obstacle_cruise_control.param.yaml" />
```
Comment (tkimura4) [inline planning_launch/launch/scenario_planning/lane_driving/motion_planning/motion_planning.launch.xml]: Is this right?

Comment (takayuki5168) [reply planning_launch/launch/scenario_planning/lane_driving/motion_planning/motion_planning.launch.xml]: https://github.com/tier4/autoware_launch/pull/330/commits/fc3fdbd05aa1ed82b2ec0601b4f603d3be76663e
I'm so sorry, I didn't check XML launch file :bow:

Comment (takayuki5168) [reply planning_launch/launch/scenario_planning/lane_driving/motion_planning/motion_planning.launch.xml]: I confirmed that there are no files including the word "obstacle_cruise_control".

**Review by tkimura4 — DISMISSED** [review]
LGTM

File: autoware_launch/rviz/autoware.rviz [hunk 3]
```diff
@@ -1332,6 +1332,18 @@ Visualization Manager:
...
+                          Enabled: true
+                          Name: VirtualWall (ObstacleCruise)
+                          Namespaces:
+                            {}
+                          Topic:
+                            Depth: 5
+                            Durability Policy: Volatile
+                            History Policy: Keep Last
+                            Reliability Policy: Reliable
+                            Value: /planning/scenario_planning/lane_driving/motion_planning/obstacle_cruise_planner/debug/stop_wall_marker
```
Comment (yukkysaito) [inline autoware_launch/rviz/autoware.rviz]:
```suggestion
                            Value: /planning/scenario_planning/lane_driving/motion_planning/obstacle_cruise_planner/virtual_wall
```

Comment (takayuki5168) [reply autoware_launch/rviz/autoware.rviz]: You mean, you want me to change the topic name?

Comment (takayuki5168) [reply autoware_launch/rviz/autoware.rviz]: @yukkysaito

Comment (yukkysaito) [reply autoware_launch/rviz/autoware.rviz]: Sorry, I'm editing the comment.
Can you please change the name to match the other virtual wall tpoic?

Comment (takayuki5168) [reply autoware_launch/rviz/autoware.rviz]: Got it

**Comment by takayuki5168** [comment]
@yukkysaito give me lgtm

**Review by yukkysaito — APPROVED** [review]
LGTM

**Pull request merged**